practice-two/src/js/script.js

```diff
@@ -1 +1,61 @@
+//QUERY TO ID
+const fullNameInput = document.getElementById('fullName');
+const emailInput = document.getElementById('email');
+const salaryInput = document.getElementById('salary');
+const cityInput = document.getElementById('city');
+//CHECK THE INPUT CONDITION
+const alphabetRegex = /^[a-zA-Z]+ [a-zA-Z]+$/;
+const emailRegex = /^([a-zA-Z0-9_\.\-])+\@(([a-zA-Z0-9\-])+\.)+([a-zA-Z0-9]{2,4})+$/;
+//FIXED VALUE
+const notification = {
+  emptyFormat: 'Empty format',
+  wrongFormat: 'Enter the wrong format. Please re-enter',
+  correctFormat: ''
```

Comment:
correctFormat should not be a const. We just show an error message when value is invalid, but in success case, we show nothing (means Clear all message).